src/js/components/Chart/index.d.ts

```diff
@@ -13,7 +13,7 @@ export interface ChartProps {
   gridArea?: GridAreaType;
   margin?: MarginType;
   bounds?: number[][];
-  color?: string | {color?: string,opacity?: "weak" | "medium" | "strong" | boolean};
+  color?: string | {color?: string,opacity?: "weak" | "medium" | "strong" | boolean} | {color?: string,value: number | number[]}[];
```

Comment:
```suggestion
  color?: string | {color?: string,opacity?: "weak" | "medium" | "strong" | boolean} | {color?: string, value?: number | number[]}[];
```

Reply:
Actually, both color and value are required if passing an array of objects. I've updated the typings to reflect this.